Frameworks/CoreText/CTRun.mm

```diff
@@ -294,12 +315,11 @@ void CTRunDraw(CTRunRef run, CGContextRef ctx, CFRange textRange) {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes Only support identity matrix
```

Comment:
I wrote it to mean "We only support identity matrix," though it could also be interpreted as "CTRun only supports identity matrix."  I suppose I should use complete sentences.